spec/features/user_adds_sight_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "spec_helper"
+
+feature "User adds sight" do
+  scenario "for authenticated user" do
+    user1 = Fabricate(:user)
+    sign_in(user1)
+    monuments = Fabricate(:category, name: "Monuments")
+    sight1 = Fabricate.attributes_for(:sight)
+
+    click_link "Add a Sight"
+    expect(page).to have_content "Create a Sight"
+
+    fill_in "Name", with: sight1[:name]
+    fill_in "Address", with: sight1[:address]
+    select  "Monuments", from: "Category"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.